### ISSUE TITLE
FEATURE REQUEST ... button "close" should NOT close ScreenToGif

### ISSUE THREAD

**Issue by user3323** [NickeManarin/ScreenToGif#262]
When you press "Record" and then rethink something and want to go back to "main screen" - you often press "Close" button at top-right. And it should return you to the previous window, whatever the window it was. It works like this literally everywhere.

Here is another action - it closes ScreenToGif at all:

![stg 1](https://user-images.githubusercontent.com/24613274/36004647-581b0bb2-0d3c-11e8-9ac2-65ecb83c6803.gif)

IMO, it has to do the same as "left arrow" button - just "go back".

//////////

The same annoying behavior is in the editor. When you click on top-right "close" button - it will close the entire ScreenToGif. But it should close only the editor.

I think everyone record their first (or maybe 2nd or 3rd) gif by wrong way. And after they made some mistakes in the recorded gif - they want to close the editor and re-record it again - and every time they should launch ScreenToGif again and again.

Please, fix this "close button" behavior. It should close only that window, to which it belongs.

Thanks.

**Comment by NickeManarin** [comment]
I agree with the first part of your request, the close button won't close the app anymore when opened from the Startup window (I removed the back button, now there's only the close button).

I don't agree with the second part, the editor should close if the close button is pressed. The editor is capable of doing the same thing that the Startup window does.

But, (with the next release) the app will provide a tray icon that will stay visible even if the last window gets closed (optional), capable of launching the recorders/editor/options via keyboard shortcuts or via a context menu.

TL;DR: Close button of the recorders will return to the Startup window.

**Comment by user3323** [comment]
@NickeManarin 

awesome! "Hide to tray" decision - is the great one!
